gemelli/_ctf_defaults.py

```diff
@@ -35,9 +35,17 @@
              " by other state inputs.")
 QORD = ("A trajectory is an ordination that can be visualized"
         "over time or another context.")
-QDIST = ("A sample-sample distance matrix generated from the euclidean distance"
-         " of the subject-state ordinations and itself.")
+QDIST = ("A sample-sample distance matrix generated from "
+         " the euclidean distance of the subject-state "
+         "ordinations and itself.")
 QLOAD = ("Compositional biplot of subjects as points and features as arrows."
-        " Where the variation between subject groupings is explained by the"
-        " log-ratio between opposing arrows. WARNING: The % variance explained"
-        " is spread over n_components and can be inflated.")
+         " Where the variation between subject groupings is explained by the"
+         " log-ratio between opposing arrows. "
+         "WARNING: The % variance explained is spread over n_components "
```

Comment:
maybe you could put something like "only spread over n_components" to indicate that this is lower than nomral

Reply:
done, thanks!